samples/sensor/dps310/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
Please add the sample in a separate commit in the same PR.

Reply:
Done